Sources/VLCMovieViewController.m

```diff
@@ -1728,6 +1773,24 @@ - (void)swipeRecognized:(UISwipeGestureRecognizer*)swipeRecognizer
     }
 }
 
+- (void)adaptMovieViewToNotch:(BOOL)adapt
+{
+    CGFloat constant = adapt ? 30.0 : 0.0;
```

Comment:
Adapt always seems to be _vpc.currentAspectRatio != VLCAspectRatioFillToScreen so you don't need the param and moving this into this function with a bool will make the code so easy to read.

Reply:
> the 30 is for I guess movieviewInset? is that supposed to be the safeareainset?

No to both questions.
We decided to cut as minimum as possible and not use the `safeAreaInset` which on landscape would be 44.0.

Using the `safeAreaInset` would be more future proof indeed, but it will go against the decision mentioned above. I will gladly adapt it if we agree to change the decision.

As for the "magic number". I don't think there is a way to retrieve this "officially"? 
Please let me know if there is a way to retrieve it, since I'm not particularly a fan myself.

Just to be clear, the "magic number" represent the cutout a.k.a. notch.

![cutout](https://i.gyazo.com/a5f197124a9307a3bc6849060a80dbed.png)

_Source: [paintcodeapp.com](https://www.paintcodeapp.com/news/iphone-x-screen-demystified)_

Reply:
I added a small comment about it, we could also make it a define/constant if you prefer.